src/algebra/pempty_instances.lean

```diff
@@ -0,0 +1,21 @@
+/-
+Copyright (c) 2021 Julian Kuelshammer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Julian Kuelshammer
+-/
+
+import algebra.module.basic
+
+/-!
+# Instances on pempty
+
+This file collects facts about algebraic structures on the (universe-polymorphic) empty type, e.g.
+that it is a semigroup.
+-/
+
+universes u
+
+@[to_additive]
+instance : semigroup (pempty) :=
```

Comment:
```suggestion
instance : semigroup (pempty.{u}) :=
```

Reply:
That didn't work, and u is also not specified in the `punit_instances`.

Reply:
Hmm, does `.{u+1}` work?

Reply:
Let's leave this for someone who actually knows how this works to review, as opposed to me just guessing ;)